layer/USAGE_android.md

```diff
@@ -1,143 +1,151 @@
+Log Detailed | debug.gfxrecon.log_detailed | BOOL | Include name and line number from the file responsible for the log message.
+Log File | debug.gfxrecon.log_file | STRING | When set, log messages will be written to a file at the specified path.
+Log File Create New | debug.gfxrecon.log_file_create_new | BOOL | Specifies that log file initialization should overwrite an existing file when true, or append to an existing file when false.
+Log File Flush After Write | debug.gfxrecon.log_file_flush_after_write | BOOL | Flush the log file to disk after each write when true.
+Log File Keep Open | debug.gfxrecon.log_file_keep_open | BOOL | Keep the log file open between log messages when true, close and reopen the log file for each message when false.
+Log Level | debug.gfxrecon.log_level | STRING | Specify the highest level message to log.  Options are: "debug", "info", "warning", "error", and "fatal".  The specified level and all levels listed after it will be enabled for logging.  For example, choosing the "warning" level will also enable the "error" and "fatal" levels.
+Log Output To Console | debug.gfxrecon.log_output_to_console | BOOL | Log messages will be written to Logcat.
+Memory Tracking Mode | debug.gfxrecon.memory_tracking_mode | STRING | Specifies the memory tracking mode to use for detecting modifications to mapped Vulkan memory objects. Available options are: "page_guard", "assisted" and "unassisted". Default is "page_guard". <ul><li>"page_guard" tracks modifications to individual memory pages, which are written to the capture file on calls to `vkFlushMappedMemoryRanges`, `vkUnmapMemory`, and `vkQueueSubmit`. Tracking modifications requires allocating shadow memory for all mapped memory.</li><li>"assisted" expects the application to call `vkFlushMappedMemoryRanges` after memory is modified; the memory ranges specified to the `vkFlushMappedMemoryRanges` call will be written to the capture file during the call.</li><li>"unassisted" writes the full content of mapped memory to the capture file on calls to `vkUnmapMemory` and `vkQueueSubmit`. It is very inefficient and may be unusable with real world applications that map large amounts of memory.</li></ul>
 
 ## Capture Files
+By default, capture files are written to `/sdcard/gfxrecon_capture.gfxr`
 
-By default, capture files are dumped out to the `/sdcard`, and will
-use the name `gfxrecon_capture.gfxr`:
+Capture files are created on the first call to vkCreateInstance, when the Vulkan
+loader loads the capture layer, and are closed on vkDestroyInstance, when the 
+last active instance is destroyed and the layer is unloaded.
 
- ```
-/sdcard/gfxrecon_capture.gfxr
-```
-
-Enabling timestamps may be needed to prevent apps that create multiple files from
-overwriting existing files.
-
-### Overriding the Default Name
+If multiple instances are created concurrently, only one capture file will be created.
+If multiple instances are created consecutively, each instance will create a new file.
```

Comment:
How can this be?  There is only one setting for the file name.  If multiple instances are created, how can we create a new file for each?